apps/demos/testing/widgets/chat/Customization.test.js

```diff
@@ -3,60 +3,65 @@ import { Selector } from 'testcafe';
 import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
 import { testScreenshot } from '../../../utils/visual-tests/helpers/theme-utils';
 
+const SELECTBOX_CLASS = 'dx-selectbox';
+const CHECKBOX_CLASS = 'dx-checkbox';
+const TEXTEDITOR_INPUT_CLASS = 'dx-texteditor-input';
+const LIST_ITEM_CLASS = 'dx-list-item';
+
 fixture('Chat.Customization')
   .page('http://localhost:8080/')
   .before(async (ctx) => {
     ctx.initialWindowSize = [900, 800];
   });
 
-runManualTest('Chat', 'Customization', ['jQuery'], (test) => {
+runManualTest('Chat', 'Customization', ['jQuery', 'React', 'Vue', 'Angular'], (test) => {
   test('Customization', async (t) => {
     const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
 
     await t
-      .click('#day-headers-format')
```

Comment:
If you add changes with nth() for the checkbox, the same issue occurs with the select box between jQuery and other demos. Be attentive and fix similar remarks in all relevant places in your PR

Reply:
[47630e4](https://github.com/DevExpress/DevExtreme/pull/28482/commits/47630e4d2f141fea2c5ffd45487debf05bd35141)